### PR TITLE
FS-3220: Migrating package from local to shared workflow

### PR DESCRIPTION
**Pull request by aaronwilliamsv1**
* Updated deploy and tag-to-release to use the shared workflows repository
* Removed package

+ref: FS-3220
+semver: minor

- [ N/A ] Unit tests and other appropriate tests added or updated
- [ N/A ] README and other documentation has been updated / added (if needed)
- [ X ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -22,9 +22,11 @@ jobs:
       E2E_PAT: ${{secrets.E2E_PAT}}
       RSA256_PUBLIC_KEY_BASE64: ${{secrets.RSA256_PUBLIC_KEY_BASE64}}
       GOV_NOTIFY_API_KEY: ${{secrets.GOV_NOTIFY_API_KEY}}
-  paketo_build:
-    permissions:
-      packages: write
-    uses: ./.github/workflows/package.yml
-    with:
-      version_to_build: $(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+    paketo_build:
```
Comment (robk-dluhc) [inline .github/workflows/deploy.yml:25]: Why the indentation change?  Isn't this on the same level as test_and_deploy?

Comment (aaronwilliamsv1) [reply .github/workflows/deploy.yml:25]: Interesting...this doesn't show up like this in the IDE 🤔 
image

**Review by robk-dluhc — APPROVED** [review]
Well, the diff doesn't include the indentation change here any more either, so all good.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (aaronwilliamsv1) [reply .github/workflows/deploy.yml]: Quick commit _message_ edit has resolved this. No idea why 🤷

**Pull request merged**